src/components/card/CardImage.tsx

```diff
@@ -4,15 +4,15 @@ import {View, StyleSheet, ImageSourcePropType} from 'react-native';
 import Image, {ImageProps} from '../image';
 import * as CardPresenter from './CardPresenter';
 import asCardChild, {asCardChildProps} from './asCardChild';
-// @ts-ignore
-import {LogService} from '../../services';
 
-// TODO: Remove omitting source after imageSource deprecation
+
+// TODO: Remove omitting source after imageSource deprecation (since it's required for Image)
 export type CardImageProps = Omit<ImageProps, 'source'> & {
   /**
    * Image source, either remote source or local. Note: for remote pass object {uri: <remote_uri_string>}
    */
   imageSource?: ImageSourcePropType;
+  source?: ImageSourcePropType; //TODO: Remove after imageSource deprecation - should take it from Image props
```

Comment:
Since ImageProps has 'sorce' as required prop, we need to omit it until we finish the deprecation, otherwise, users will get an error. This is why 'source' was added as an optional prop. Together with the removal of 'imageSource' prop we should remove the optional source prop, and the omit, and let the required 'source' prop be taken from the ImageProps.

Reply:
I see